src/combined_event.rs

```diff
@@ -28,6 +30,10 @@ impl<'de> Deserialize<'de> for CombinedEvent {
                 Ok(CombinedEvent::Sns(
                     SnsEvent::deserialize(raw_value).map_err(de::Error::custom)?,
                 ))
+            } else if records[0].get("body").is_some() {
```

Comment:
Checking if "eventSource" is"aws:sqs" would be a more reliable method of recognising SQS events.

Reply:
solved.